src/com/CommunicateMesh.cpp

```diff
@@ -201,14 +199,15 @@ void CommunicateMesh::broadcastSendMesh(const mesh::Mesh &mesh)
     //contrary to the normal sendMesh, this variant must also work for adding delta meshes
     std::vector<int> vertexIDs(numberOfVertices);
     for (int i = 0; i < numberOfVertices; i++) {
-      vertexIDs[i] = mesh.vertices()[i].getID();
+      vertexIDs[i] = meshVertices[i].getID();
     }
     _communication->broadcast(vertexIDs);
 
     std::vector<int> edgeIDs(numberOfEdges * 2);
+    const auto &     meshEdges = mesh.edges();
     for (int i = 0; i < numberOfEdges; i++) {
-      edgeIDs[i * 2]     = mesh.edges()[i].vertex(0).getID();
-      edgeIDs[i * 2 + 1] = mesh.edges()[i].vertex(1).getID();
+      edgeIDs[i * 2]     = meshEdges[i].vertex(0).getID();
+      edgeIDs[i * 2 + 1] = meshEdges[i].vertex(1).getID();
```

Comment:
I know this is not your crime, but we should maybe choose a named variable for 2 (I guess `vertexIdsPerEdge` or so). `clang` complains about this 'magic' numbers and it would make the code for newcomer better readable.

Reply:
Do you mind opening an issue regarding this? That will make it easier to discuss alternatives.